libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -70,6 +71,7 @@
 #define HAL_INS_QFLIGHT  15
 #define HAL_INS_QURT     16
 #define HAL_INS_BBBMINI  17
+#define HAL_INS_AERO 18
```

Comment:
Hm, now I see why I did it like that, the other ones aren't either hehe

This is just my opinion, but I don't think it's worth trying to keep things like that aligned. When a new macro with a bigger name is added we need to align everyone again. If that is done in the same commit, then it doesn't highlight to the reader what's been added; if in a separate commit, then we have another patch.

Anyway, I've added a patch reformatting these and other defines, will push soon with the other fixes.